cloudwatch/vespa_cloudwatch_emitter.py

```diff
@@ -14,6 +14,9 @@ class VespaCloudwatchEmitter:
 
     VESPA_URL = 'http://my-host:8080/metrics/v2/values'
     NAMESPACE = 'my-cloudwatch-namespace'
+    KEY_NAME = 'album-recommendation-private-key'
+    CERT_NAME = 'album-recommendation-public-cert'
+    SSM_REGION = "us-east-1"
     CHUNK_SIZE = 20
```

Comment:
We might want to have these as env variables instead, so the tenant can just copy-paste this code, and configure everything directly in the AWS console.